src/llmq/blockprocessor.cpp

```diff
@@ -96,18 +96,18 @@ void CQuorumBlockProcessor::ProcessMessage(const CNode& peer, std::string_view m
             // same, can't punish
             return;
         }
-        int quorumHeight = pQuorumBaseBlockIndex->nHeight - (pQuorumBaseBlockIndex->nHeight % llmq_params_opt->dkgInterval) + int(qc.quorumIndex);
-        if (quorumHeight != pQuorumBaseBlockIndex->nHeight) {
+        if (int quorumHeight = pQuorumBaseBlockIndex->nHeight - (pQuorumBaseBlockIndex->nHeight % llmq_params_opt->dkgInterval) + int(qc.quorumIndex);
```

Comment:
I think we should ignore this rule in your sanitizer. `quorumheight` is small stack variable, but this `if` become a monster; and that's not a single example